versions-maven-plugin/src/it-repo/dummy-aggregate-child-333.pom

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>localhost</groupId>
+    <artifactId>dummy-aggregate-parent</artifactId>
+    <version>1.0.1-SNAPSHOT</version>
+  </parent>
+
+  <groupId>localhost</groupId>
+  <artifactId>dummy-aggregate-child</artifactId>
+  <version>1.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <properties>
+    <version.maven-clean-plugin>2.1</version.maven-clean-plugin>
+    <version.maven-deploy-plugin>2.1</version.maven-deploy-plugin>
```

Comment:
Can we use a little newer versions in test or a dummy dependency

Reply:
Using newer version of `maven-clean-plugin`, also revisited and changed test poms, so they mainly use dummy dependencies.